src/web/mjs/connectors/BlogTruyen.mjs

```diff
@@ -32,11 +33,12 @@ export default class BlogTruyen extends Connector {
     }
 
     async _getMangasFromPage(page) {
-        let uri = new URL('/ajax/Search/AjaxLoadListManga?key=tatca&orderBy=1&p=4', this.url);
+        let uri = new URL('/ajax/Search/AjaxLoadListManga', this.url);
         uri.searchParams.set('key', 'tatca'); // show all
         uri.searchParams.set('orderBy', 1); // order by title
         uri.searchParams.set('p', page);
         let request = new Request(uri, this.requestOptions);
+        request.headers.set('X-Requested-With', 'XMLHttpRequest');
```

Comment:
Can't this be done in this.requestOptions in initialize ?

Reply:
I'd rather not. It would put the header on all requests when its just need for this one.